mmv1/products/compute/api.yaml

```diff
@@ -8333,7 +8333,7 @@ objects:
     name: 'NodeGroup'
     kind: 'compute#NodeGroup'
     base_url: projects/{{project}}/zones/{{zone}}/nodeGroups
-    create_url: projects/{{project}}/zones/{{zone}}/nodeGroups?initialNodeCount={{size}}
+    create_url: projects/{{project}}/zones/{{zone}}/nodeGroups?initialNodeCount=PRE_CREATE_REPLACE_ME
```

Comment:
I think we'll need to make most of these changes in `terraform.yaml` (any api.yaml property can be used there, it overwrites the api.yaml value) to avoid breaking Ansible / possibly InSpec